xlt-timerrecorder-chrome/src/background.js

```diff
@@ -1,21 +1,60 @@
+// log something so we can see when a service worker was (re-)activated
+log("Service worker activated");
+
+// send a keep-alive message every 10 secs to prevent the service worker from being deactivated after 30 secs of inactivity
+setInterval(
+  () => {
```

Comment:
Just for consistency reasons. We may rework entire code base to use modern ECMAScript features such as arrow functions, async/await, destructuring assignment etc. later.
 
```suggestion
  function ping() {
```